sequence/arguments/api.graphql

```diff
@@ -57,23 +60,21 @@ type Query {
             }
             """
         )
-    # A sequence call that sequences converCityOrLocaleToLatLong and weather
-    # this allows for a cityOrLocale argument to be used to fetch weather data by lat/lon
-    # we have to use an arguments setting for the second query, because   
-    # query arguments only flow to the first query in the sequence
-    # but we need units to flow to the second query
-    # we are saying that the argument named units in the query weather 
-    # gets its value from the argument units in the enclosing query
-    # In addition to the arguments explictly passed down to the second query,
-    # it also gets the output of the first query
-    # in this case, the first query returns a type Coord, and therefore its fields lat and lon are
-    # available to the second query.
+    # Below is a sequence call that sequences `convertCityOrLocaleToLatLong` and `weather`.
+    # Now the `cityOrLocale` argument can fetch weather data by lat/lon.
+    # We have to use an `arguments` setting for the second query, because   
+    # enclosing query arguments are only directly accessible from the first query in the sequence,
```

Comment:
"enclosing query arguments are only directly accessible from the first query in the sequence,"
Wrong way around, not sure it's relevant in this discussion what the first step is doing though.

Reply:
made updates in response to these pieces of feedback